### PR TITLE
feat: add zoom-in zoom-out cursor

### PR DESCRIPTION
**Pull request by naoyashiga**
##### Checklist

- [x] `npm test` passes
- [x] tests and/or benchmarks are included
- [x] commit message follows commit guidelines

##### Description of change

You can use zoomIn and zoomOut cursor in register-interaction. 

This is useful. For example, you use scale-transform zooming.
```js
registerInteraction('custom-zoom', {
  showEnable: [
    { trigger: 'plot:mouseenter', action: 'cursor:zoomIn' },
    { trigger: 'plot:mouseleave', action: 'cursor:zoomOut' },
  ],
});
```

### CODE REVIEW

**Comment by naoyashiga** [comment]
It may be strange chinese explanation in documents because I'm not native Chinese speaker.

```
光标显示可以被放大
光标显示可以缩小尺寸
```

If it needs modification, please tell me correct explations.

File: tests/unit/interaction/action/cursor-spec.ts [hunk 1]
```diff
@@ -28,6 +28,7 @@ describe('active test', () => {
   first.shape.attr('cursor', 'pointer');
   const action = new CursorActive(context);
   const cursors = ['default', 'crosshair', 'pointer', 'move', 'wait', 'help', 'text'];
+  const zoomCursors = ['In', 'Out'];
```
Comment (hustcc) [inline tests/unit/interaction/action/cursor-spec.ts:31]: 直接放到 cursors 数组中可以吗？

Comment (naoyashiga) [reply tests/unit/interaction/action/cursor-spec.ts:31]: If I add zoomIn and zoomOut to cursors, I need to logics for zoom in cursors each loop.
https://github.com/antvis/G2/blob/67e80988d8e848bb1e336402fd7e9ac4498a42b5/tests/unit/interaction/action/cursor-spec.ts#L37-L40

resizeArr is not in cursors and have different test code （'test resize cursors'）. I wrote zoomCursors like this.

Comment (hustcc) [reply tests/unit/interaction/action/cursor-spec.ts:31]: > const  cursors = ['default', 'crosshair', 'pointer', 'move', 'wait', 'help', 'text', 'zoom-in', 'zoom-out'];

**Review by hustcc — CHANGES_REQUESTED** [review]

**Comment by hustcc** [comment]
Good job, in the test case, you can combine `cursors` and `zoomCursors ` into only one array.

File: tests/unit/interaction/action/cursor-spec.ts (hunk 1 above)
Comment (visiky) [reply tests/unit/interaction/action/cursor-spec.ts:31]: no need to add this variable, just add the test caes like:
```ts
 it('zoom cursor', () => {
    each(['zoom-in', 'zoom-out'], (cursor: string) => {
     const method  = cursor.split('-')[1];
      action[`zoom${method.toUpperCase()}`]();
      expect(canvas.getCursor()).toBe(cursor);
    });
  });
```

**Pull request merged**